iothub/device/src/Transport/Amqp/AmqpUnit.cs

```diff
@@ -123,30 +131,27 @@ internal async Task<AmqpIotSession> EnsureSessionIsOpenAsync(CancellationToken c
 
             try
             {
-                if (_amqpIotSession == null
-                    || _amqpIotSession.IsClosing())
+                if (_amqpIotSession == null || _amqpIotSession.IsClosing())
                 {
                     _amqpIotSession?.SafeClose();
 
-                    _amqpIotSession = await _amqpConnectionHolder
-                        .OpenSessionAsync(_deviceIdentity, cancellationToken)
-                        .ConfigureAwait(false);
+                    _amqpIotSession = await _amqpConnectionHolder.OpenSessionAsync(_deviceIdentity, cancellationToken).ConfigureAwait(false);
+
```

Comment:
I've updated these lines to break only if they have >=3 args being passed into the method call. Otherwise, it seemed to me like we were breaking too soon and that was decreasing the readability.

Let me know if you feel the previous indentation was more readable. I don't feel too strongly about it; this just made it more readable for me.

Reply:
To me it is mostly about line length, but also when exceeding 3 args.

Reply:
For this case, I can see the full line in diff split view on my work monitor, so it doesn't require line splitting for the first reason, either.